core/templates/pages/collection-editor-page/services/collection-editor-state.service.ts

```diff
@@ -20,26 +20,24 @@
 
 require('domain/collection/CollectionObjectFactory.ts');
 require('domain/collection/collection-rights-backend-api.service.ts');
-require('domain/collection/CollectionRightsObjectFactory.ts');
 require('domain/collection/editable-collection-backend-api.service.ts');
 require('domain/editor/undo_redo/undo-redo.service.ts');
 require('services/alerts.service.ts');
-
 require('pages/collection-editor-page/collection-editor-page.constants.ajs.ts');
 
 import { EventEmitter } from '@angular/core';
 
+import { CollectionRights } from 'domain/collection/CollectionRightsObjectFactory';
 angular.module('oppia').factory('CollectionEditorStateService', [
   '$rootScope', 'AlertsService', 'CollectionObjectFactory',
-  'CollectionRightsBackendApiService', 'CollectionRightsObjectFactory',
+  'CollectionRightsBackendApiService',
   'EditableCollectionBackendApiService', 'UndoRedoService',
   function(
       $rootScope, AlertsService, CollectionObjectFactory,
-      CollectionRightsBackendApiService, CollectionRightsObjectFactory,
+      CollectionRightsBackendApiService,
       EditableCollectionBackendApiService, UndoRedoService) {
     var _collection = CollectionObjectFactory.createEmptyCollection();
-    var _collectionRights = (
-      CollectionRightsObjectFactory.createEmptyCollectionRights());
```

Comment:
@kevintab95, as you see I have removed the initialization of the _collectionRights, calling this variable without initializing it, will give undefined, the earlier spec file was made in accordance with this initialization.  The initialization is removed because the createEmptyCollectionRights() is no more there. @nishantwrp , please add if I missed something.

Reply:
Thanks for explaining!